setupTests.ts

```diff
@@ -0,0 +1,5 @@
+import 'jest-enzyme';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-preact-pure';
+
+configure({ adapter: new Adapter });
```

Comment:
Can we do something about "ESLint: Unsafe call of an `any` typed value"?
Also newline is missing at end of file.

Reply:
Done